kite-data/kite-data-core/src/main/java/org/kitesdk/data/Datasets.java

```diff
@@ -131,17 +172,32 @@
   }
 
   /**
-   * Create a {@link Dataset} for the given dataset or view URI.
+   * Create a {@link Dataset} for the given dataset or view URI string.
```

Comment:
In this case, the URI must be an instance of java.net.URI, not String, so the addition of "string" should be reverted.